postcss/scope.js

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/**
+ * This custom plugin prepends every css rule in the generated style.css
+ * with an extra selector, essentially scoping the whole css file to the specific block
+ */
+
+/**
+ * @type {import('postcss').PluginCreator}
+ */
+module.exports = (opts = {}) => {
+    return {
+        postcssPlugin: "scope",
+        Root(root) {
+            root.walkRules((rule) => {
+                if (
+                    rule.parent &&
+                    rule.parent.type === "atrule" &&
+                    rule.parent.name.includes("keyframes")
+                ) {
+                    return;
+                }
+                rule.selectors = rule.selectors.map((selectors) => {
+                    return selectors.split(/,\s* /g).map((selector) => {
+                        if (selector === opts.selector) {
+                            return selector;
+                        }
+                        const newSelector = `${opts.scope} ${selector}`;
+                        return newSelector;
+                    });
+                });
```

Comment:
```suggestion
               rule.selectors = rule.selectors.map(originalSelector =>
                    originalSelector.split(/,\s*/g).map(individualSelector => {
                        return individualSelector === opts.selector ? individualSelector : `${opts.scope} ${individualSelector}`;
                    }).join(", ") // Joining back individual selectors into a combined form
                );
```

Comment is just for clarification

Reply:
It was actually automatically joining the individual selectors in the end, but I added the join(", ") anyway to make sure. Thanks!